torch/_inductor/graph.py

```diff
@@ -670,11 +672,12 @@ def output(self, target, args, kwargs):
                     pass
 
         self.finalize()
-        log.debug(
-            "Force channels last inputs for %d conv for the current graph with id %d",
-            self.num_channels_last_conv,
-            self.graph_id,
-        )
+        if inductor_config.conv_force_channels_last:
+            log.debug(
```

Comment:
we still want the same debug logic to be printed even if the newly added config option is False